rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -440,14 +437,20 @@
       ],
       "properties": {
         "$ref": {
+          "title": "Vendor URI",
           "type": "string",
           "pattern": "^https://ils.rero.ch/api/vendors/.*?$",
           "form": {
+            "focus": true,
```

Comment:
Are you sure to want that the vendor field has the main focus in the editor ?

Reply:
bien vu